### PR TITLE
GT 270 - Beginnings of the "Tract" Controller

### PR DESCRIPTION
**Pull request by ryancarlson**
Adds an initial empty view controller that will render the tracts.

This PR includes setting up the navigation bar option with some stub methods to be filled in with real data and logic later.

Also in this PR:
 - removes Reachability checking b/c Apple advises against it.
 - extract some more strings to localized folder
 - remove error alerts when loading from disk.

### CODE REVIEW

**Comment by ryancarlson** [comment]
Hey @pablomarti here is some initial logic for the view controller that will render the tracts/booklets. I included a couple other things I noticed along the way

**Comment by pablomarti** [comment]
Ok great @ryancarlson ! Will check it

File: godtools/FlowControllers/PlatformFlowController.swift [hunk 1]
```diff
@@ -37,6 +37,11 @@ HomeViewControllerDelegate, ToolDetailViewControllerDelegate, AddToolsViewContro
         self.pushViewController(viewController: viewController)
     }
     
+    func moveToTract() {
+        let viewController = TractViewController(nibName: String(describing: TractViewController.self), bundle: nil)
```
Comment (pablomarti) [inline godtools/FlowControllers/PlatformFlowController.swift]: Good one!

File: godtools/ViewControllers/Platform/HomeViewController.swift [hunk 2]
```diff
@@ -85,10 +86,10 @@ class HomeViewController: BaseViewController, ToolsManagerDelegate {
...
     }
-    
-    // MARK: - ToolsManagerDelegate
-    
+}
+
+extension HomeViewController: ToolsManagerDelegate {
     func didSelectTableViewRow(cell: HomeToolTableViewCell) {
-        self.delegate?.moveToToolDetail()
+        self.delegate?.moveToTract()
```
Comment (pablomarti) [inline godtools/ViewControllers/Platform/HomeViewController.swift]: 👍

File: godtools/ViewControllers/Platform/LanguagesTableViewController.swift [hunk 3]
```diff
@@ -39,16 +39,16 @@ class LanguagesTableViewController: BaseViewController {
     // MARK: - Load data
     
     func loadFromDisk() {
-        languagesManager.loadFromDisk().catch(execute: { error in
-            self.showAlertControllerWith(message: error.localizedDescription)
-        }).always {
+        languagesManager.loadFromDisk().always {
```
Comment (pablomarti) [inline godtools/ViewControllers/Platform/LanguagesTableViewController.swift]: Great, this looks clean

File: godtools/ViewControllers/Tract/TractViewController.swift [hunk 4]
```diff
@@ -0,0 +1,68 @@
+//
```
Comment (pablomarti) [inline godtools/ViewControllers/Tract/TractViewController.swift]: This looks great, I was going to tell you to use the parent method ` displayScreenTitle()` but this is a special case, maybe this viewcontroller will requiere changes during the development

File: godtools/ViewControllers/Tract/TractViewController.swift [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+    fileprivate func displayTitle() {
+        if parallelLanguageIsAvailable() {
+            navigationItem.titleView = languageSegmentedControl()
+        } else {
+            navigationItem.title = currentTractTitle()
+        }
+    }
+    
+    fileprivate func createHomeButton() -> UIBarButtonItem {
```
Comment (pablomarti) [inline godtools/ViewControllers/Tract/TractViewController.swift]: Can you move this methods to BaseViewController? Please see how the buttons are created and loaded there, that will help us to reuse the buttons across viewcontrollers

Comment (ryancarlson) [inline godtools/ViewControllers/Tract/TractViewController.swift]: Okay, yes I will look at this

**Comment by ryancarlson** [comment]
@pablomarti I just pushed a commit. is that what you're going for?

**Comment by pablomarti** [comment]
Great @ryancarlson feel free to merge!

**Comment by ryancarlson** [comment]
thanks for the review!

**Pull request merged**